extensions/resource-deployment/src/interfaces.ts

```diff
@@ -202,9 +234,13 @@ export enum FieldType {
 	SQLPassword = 'sql_password',
 	Password = 'password',
 	Options = 'options',
+	RadioOptions = 'radio_options',
 	ReadonlyText = 'readonly_text',
 	Checkbox = 'checkbox',
-	AzureAccount = 'azure_account'
+	AzureAccount = 'azure_account',
+	AzureLocations = 'azure_locations',
+	FilePicker = 'file_picker',
+	KubeConfigClusterPicker = 'kube_config_cluster_picker'
```

Comment:
we are picking cluster context, not cluster.

Reply:
yes indeed. Will rename.